Makefile

```diff
@@ -59,3 +62,10 @@ helm-docs: $(HELM_DOCS)
 ngc-helm-docs: download-ngc-helm-chart helm-docs
 
 branch-helm-docs: download-branch-helm-chart helm-docs
+
+.PHONY: generate-docs-versions-var
+generate-docs-versions-var:
+	cd hack && curl -sL ${BRANCH_RELEASE_YAML_URL} -o release.yaml
+	cd hack && $(GO) run release.go --templateDir ./templates/ --outputDir .
```

Comment:
can outputDIr point to docs/common/vars.rst ?

Reply:
The output is with yaml suffix, so I need to rename anyway

Reply:
i see, can you use a path under ./build/ as --outputDir for template rendering ? feels more tidy :)